.github/workflows/wait-for-connection-test.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
     timeout-minutes: 60
     container:
       # TODO: Needs newer, light weight image
```

Comment:
Please remove (This is the lighter weight image)

Reply:
Remove the todo specifically

Reply:
Thanks! I removed it.